DXRModules/DeusEx/Classes/DXREventsBase.uc

```diff
@@ -1414,8 +1417,8 @@ static function int BingoActiveMission(int currentMission, int missionsMask)
     if(missionAnded != 0) return 2;// 2==true
 #endif
 
-    if(missionsMask < (1<<minMission)) {
```

Comment:
removing the check for mission number here and fully relying on FAILED_MISSION_MASK (set by CheckForExpiredBingoGoals) means there's a circular dependency?

CheckForExpiredBingoGoals depends on BingoActiveMission and now BingoActiveMission depends on CheckForExpiredBingoGoals since it is no longer checking mission number itself?

Reply:
You're right. Expired goals weren't getting marked correctly.